OpenMTR/Utilities/ImageUtils.cs

```diff
@@ -34,5 +34,10 @@ public static void ApplyGaussianBlur(List<DataObject> dataObjectList)
                 Cv2.GaussianBlur(dataObject.SourceImage, dataObject.ModifiedImage, new Size(3, 3), 0, 0, BorderTypes.Default);
             }
         }
+
+        public static Mat GetKernel(int size = 3)
```

Comment:
I didn't realize that in the newer version of the API it didn't require a Mat Object. Nifty. If we are going to pass in a variables to this I recommend allowing them to be set instead of hard coding them.